sormas-e2e-tests/src/test/resources/features/sanity/web/EpidemiologicalDataCase.feature

```diff
@@ -163,4 +163,24 @@ Feature: Epidemiological data coverage
     And I click on the NEW CONTACT button in in Exposure for Epidemiological data tab in Cases
     And I click on the CHOOSE CASE button in Create new contact form in Exposure for Epidemiological data tab in Cases
     And I search and chose the last case uuid created via UI in the CHOOSE CASE Contact window
-    And I click on SAVE button in create contact form
\ No newline at end of file
+    And I click on SAVE button in create contact form
+
+  @issue=SORDEV-5525 @env_de
+  Scenario: Enter an activity as case in Epidemiological data tab in Cases for DE version
+    Given I log in with National User
+    And I click on the Cases button from navbar
+    And I click on the NEW CASE button
+    When I create a new case with specific data for DE version
+    And I check case created from created contact is correctly displayed on Edit Case page for DE
+    And I navigate to case person tab
+    And I navigate to epidemiological data tab in Edit case page
+    Then I click on Activity details known with UNBEKANNT option
```

Comment:
We should not use non-english words in the step definition (even as parameters).
It's better to write something like "I click the German option for unknown", and then have it pick the "unbekannt" string from within the step logic.
Same applies to all the other German words below.